src/app/market/features/AgentDetailContent/AgentInfo/Header.tsx

```diff
@@ -22,6 +24,21 @@ const Header = memo(() => {
   const { meta, createAt, author, homepage, config } = agentItem;
   const { avatar, title, description, tags, backgroundColor } = meta;
 
+  const handleAddAgentAndConverse = async () => {
+    if (!agentItem) return;
+
+    const session = await createSession({ config, meta });
+    message.success(t('addAgentSuccess'));
+    router.push(`/chat?session=${session}`);
```

Comment:
please use `SESSION_CHAT_URL` to adopt the switch of mobile and desktop:

https://github.com/lobehub/lobe-chat/blob/f0a3cbad488b03e1d96fdeb854c87e228266d89a/src/const/url.ts#L37-L38

```ts
  const isMobile = useIsMobile();

    router.push(SESSION_CHAT_URL(session,isMobile));
```